fsevents.js

```diff
@@ -6,7 +6,11 @@
 /* jshint node:true */
 'use strict';
 
-var Native = require('./build/Release/fse');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var Native = require(binding_path);
```

Comment:
@es128 was originally directly copied from the example. I agree it can be simplified - and I should have noticed it before :-)

`binary.find('package.json')` wont work since when require is called it's would be relative to pre-binding.js

Expect a change shortly.